jsonnet/telemeter/client.libsonnet

```diff
@@ -6,5 +6,10 @@
     telemeterClient+: {
       matchRules+: [],
     },
+
+    commonLabels+:: {
+      'app.kubernetes.io/part-of': 'openshift-monitoring',
```

Comment:
This field/object doesn't seem to be used anywhere in the kubernetes.libsonnet code. I think you need to either manually add it here [0] or use the `commonLabels` field in that file somewhere. 

[0[ https://github.com/openshift/telemeter/blob/master/jsonnet/telemeter/client/kubernetes.libsonnet#L95

Reply:
👍